### DISCUSSION TITLE
Black rendering artefacts with normal maps

### DISCUSSION THREAD

**Discussion by georgy95** [mitsuba-renderer/mitsuba3#918] [Q&A] [answered]
I'm rendering a sample scene with Principled BSDF material - black artefacts appear around the object.

![Group 7](https://github.com/mitsuba-renderer/mitsuba3/assets/6961787/e2c99af0-9b44-4f75-bb8e-3d6bda7a519a)

I [raised the following issue](https://github.com/mitsuba-renderer/mitsuba3/issues/919).

**Comment by njroussel** [comment, marked as answer]
Tracked in #919.
